app/controllers/miq_task_controller.rb

```diff
@@ -394,32 +385,40 @@ def build_query_for_warn
   end
 
   def build_query_for_status_completed(status)
-    return ["(state=? AND status=?)", ["finished", status]] if vm_analysis_task?
-    ["(state=? AND status=?)", ["Finished", status.capitalize]]
+    return ["(#{db_table}state=? AND #{db_table}status=?)", ["finished", status]] if vm_analysis_task?
+    ["(#{db_table}state=? AND #{db_table}status=?)", ["Finished", status.capitalize]]
   end
 
   def build_query_for_running
-    return ["(state!=? AND state!=? AND state!=?)", %w(finished waiting_to_start queued)] if vm_analysis_task?
-    ["(state!=? AND state!=? AND state!=?)", %w(Finished waiting_to_start Queued)]
+    sql = "(#{db_table}state!=? AND #{db_table}state!=? AND #{db_table}state!=?)"
+    if vm_analysis_task?
+      [sql, %w(finished waiting_to_start queued)]
+    else
+      [sql, %w(Finished waiting_to_start Queued)]
```

Comment:
I don't understand the backend part, so I need to ask: why the upper case / lower case difference?